src/_common/content/content-editor/content-editor.ts

```diff
@@ -352,7 +354,15 @@ export default class AppContentEditor extends Vue implements ContentOwner {
 				doc: Node.fromJSON(this.schema, jsonObj),
 				plugins: this.plugins,
 			});
-			this.createView(state);
+
+			const view = this.createView(state);
+
+			// Set selection at the end of the document.
+			const tr = view.state.tr;
+			const selection = Selection.atEnd(view.state.doc);
+			tr.setSelection(selection);
+			tr.scrollIntoView();
+			view.dispatch(tr);
```

Comment:
This is currently only working for messages that don't contain uploaded media or tenor gifs - I'm assuming because the media is only taking up the equivalent of a paragraph when we're scrolling the selection into view. Setting the function as `async` and adding `await this.nextTick()` here seems to fix the issue though.
```suggestion
			tr.setSelection(selection);
			await this.nextTick();
			tr.scrollIntoView();
			view.dispatch(tr);
```